CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Release 5.5.0
+ * Update to VC-K 5.5.0: Use OID4VCI Draft 15 for loading credentials
+ * Please migrate to issuing service at <https://wallet.a-sit.at/m6>
+
```

Comment:
Can someone confirm that this upgrade (with biometric auth and all) works well as a direct upgrade?

On Android, uninstall and reinstall clears everything, but in iOS, I am not so sure. I'd rather have this upgrade tested